### PR TITLE
updating onbefore event and gettargetranges

### PR DESCRIPTION
**Pull request by Rumyra**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

beforeinput and getTargetRanges was shipped in Firefox 83, so updating the information regarding both of these.

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

See https://bugzilla.mozilla.org/show_bug.cgi?id=1609291 and https://bugzilla.mozilla.org/show_bug.cgi?id=1658702

- [x] Data: if you tested something, describe how you tested with details like browser and version

Wrote a code example and works as expected in Firefox

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
Done

### CODE REVIEW

File: api/InputEvent.json [hunk 1]
```diff
@@ -206,10 +206,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "83"
```
Comment (foolip) [inline api/InputEvent.json:209]: Confirmed by running http://mdn-bcd-collector.appspot.com/tests/api/InputEvent/getTargetRanges in Firefox 82 and 83.

**Review by foolip — COMMENTED** [review]
Looks good, but I'd like @ddbeck to review as it changes a bunch of experimental flags (https://github.com/mdn/browser-compat-data/issues/6905) and also because Firefox 83 is still in beta.

**Comment by Rumyra** [comment]
Feel free to push back - this is my first BCD pr ☺️

FYI I removed experimental due to the coverage of the feature now it's added to ff 83

**Review by ddbeck — APPROVED** [review]
This looks good to me. Given the age of the implementations, I feel confident this will be swept up by the eventual guideline we settle on for "experimental". For the feature shipping in 83, I'd be very surprised if it didn't make stable, since that's coming up very soon.

Thank you! 🎉

**Comment by ddbeck** [comment]
Also, thank you for this PR, @Rumyra, and welcome to BCD!

**Comment by Rumyra** [comment]
Thanks both!

**Comment by makotokato** [comment]
This isn't shipped by release version yet, so this PR is incorrect

**Pull request merged**